dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithLksg.cy.ts

```diff
@@ -59,22 +55,21 @@ describe("Component test for the LksgPanel", () => {
 
   it("Validate that certificate links are displayed correctly", () => {
     const preparedFixture = getPreparedFixture("one-lksg-data-set-with-two-production-sites", preparedFixtures);
-    mountMLDTFrameworkPanelFromFakeFixture(DataTypeEnum.Lksg, lksgDisplayConfiguration, [preparedFixture]);
+    mountMLDTFrameworkPanelFromFakeFixture(DataTypeEnum.Lksg, lksgViewConfiguration, [preparedFixture]);
 
     getSectionHead("Governance").should("have.attr", "data-section-expanded", "false").click();
     getSectionHead("Certifications, policies and responsibilities")
       .should("have.attr", "data-section-expanded", "false")
       .click();
-
-    getCellValueContainer("SA8000 Certification").find("i[data-test=download-icon]").should("be.visible");
   });
 
   it("Validate that the list of production sites is displayed modal is displayed correctly", () => {
     const preparedFixture = getPreparedFixture("one-lksg-data-set-with-two-production-sites", preparedFixtures);
-    mountMLDTFrameworkPanelFromFakeFixture(DataTypeEnum.Lksg, lksgDisplayConfiguration, [preparedFixture]);
+    mountMLDTFrameworkPanelFromFakeFixture(DataTypeEnum.Lksg, lksgViewConfiguration, [preparedFixture]);
     const lksgData = preparedFixture.t;
-
-    cy.get(`span.p-column-title`).should("contain.text", lksgData.general.masterData.dataDate.substring(0, 4));
+    getCellValueContainer("Data Date")
```

Comment:
Das ist aber eine Änderun in Funktion. Vorher hat das glaube ich nach den reporting periods in den table heandern geschaut. Jetzt auf den data date eintrag im dataset. Why?

Reply:
Ja so kann man es auch sehen. Mir war nicht klar was hier getestet werden soll. Ob die Header oder das Data Date getestet werden soll. So oder so macht es wenig Sinn die reporting Period mit dem Data Date zu vergleichen, da die beiden Werte unabhängig voneinander in den fake fixtures generiert werden

Reply:
Test auch auf die Header ausdehnen